packages/web/src/scss/components/Button/_Button.scss

```diff
@@ -42,19 +49,19 @@
     );
 
     flex: none;
-    padding-left: 0;
-    padding-right: 0;
+    padding-inline: 0;
 }
 
 .Button--block {
```

Comment:
Could you swap this with `.Button--square` above so both `.Button--square`s are next to each other?